persistence/sql/sqlsaga.md

```diff
@@ -0,0 +1,47 @@
+---
```

Comment:
Based on the recent discussion, I understood that we're going to support both SqlSaga and core saga as equally good options, until things change (e.g. core catches up with mapping api etc.).
So based on that assumption I think we should describe both on the same page instead of moving SQL Saga away, make two sections on the same level "Saga" and "SqlSaga" (in that order), and move the SqlSaga-specific sections one level deeper in the page.

> Which API to use is largely a matter of personal preference.

IMO that'll be more confusing for the user, so not lowering the entry barrier, which is our goal here. We need a specific recommendation.

I think it's ok to say "go with core saga by default for new projects" and then say when to use SQL saga, i.e. edge cases such as derived classes for IL inspection, preference for more concise syntax, etc. But it's our thing to guide user in making that decision, especially a person new to NServiceBus.

Reply:
Made some changes that you suggested, however it's nearly impossible to present it all together on one page. I tried. The weight of the old versions (especially v1-2) makes it really difficult. I had to go with presenting the Core API as default because that's what a new visitor would be aware of, and presenting SqlSaga as another thing you could go look at with its own page. The separate page makes a different pool with lower overall complexity making the versioning possible to pull off there.